### DISCUSSION TITLE
How to make an extension have highlighting of another language

### DISCUSSION THREAD

**Discussion by nikivdev** [zed-industries/zed#21326] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I want to have `.rss` extension have rust syntax highlighting. 

How can I do this?

**Comment by dream-dasher** [comment]
[Configuring Languages: File Associations](https://zed.dev/docs/configuring-languages#file-associations)

**Comment by jnptk** [comment]
This should work:
settings.json
```json
{
    "file_types": {
        "RUST": ["*.rss"]
    }
}
```
